### PR TITLE
Adding a 'move' option to the importer's CLI.

### PR DESCRIPTION
**Pull request by karpinski**
#2252

### CODE REVIEW

**Comment by jackwilsdon** [comment]
Thanks for the PR @karpinski! Looks good to me, could you add a changelog entry?

**Comment by karpinski** [comment]
Done

File: docs/changelog.rst [hunk 1]
```diff
@@ -22,6 +22,8 @@ New features:
   value.  The argument can be any two characters that represent the left and
   right brackets. It defaults to `[]` and can also be blank to turn off
   bracketing. :bug:`2397` :bug:`2399`
+* Added a ``--move`` or ``-m`` option to the importer so that the files can be
+  moved to the library instead of being copied or added "in place".
```
Comment (jackwilsdon) [inline docs/changelog.rst]: Would it be possible to add a reference to both #2252 and this PR (#2429) to the entry?

You can add it by putting this at the end of the line: `` :bug:`2252` :bug:`2429` ``

Thanks again! :grin:

Comment (karpinski) [inline docs/changelog.rst]: Done

**Comment by jackwilsdon** [comment]
Looks good to me, what do you think @sampsyo?

**Comment by sampsyo** [comment]
This looks perfect. Thank you, @karpinski! And thanks to @jackwilsdon for reviewing.

**Pull request merged**